documentation/src/main/wiki/AETIn10Minutes.md

```diff
@@ -8,19 +8,19 @@ Before start make sure that you have enough memory on your machine (8 GB is mini
 You need to download and install the following software:
    * [VirtualBox 5.1.30](https://www.virtualbox.org/wiki/Download_Old_Builds_5_1)
    * [Vagrant 1.9.2](https://releases.hashicorp.com/vagrant/)
-   * [ChefDK 1.1.16](https://downloads.chef.io/chef-dk/)
+   * [ChefDK 2.4.17](https://downloads.chef.io/chef-dk/)
```

Comment:
There is one more place in AET where this versions are listed:
https://github.com/Cognifide/aet/blob/master/vagrant/README.md#installation

Please update it also there.
Maybe even better option is to update `/vagrant/README.md` file to refer this wiki page? To keep all versions in one place.

Reply:
I've simply updated the version to keep the changes minimal.